src/__experimental__/components/input/input-sizes.style.js

```diff
@@ -21,3 +21,14 @@ export default {
     tooltipHorizontalOffset: 6,
   },
 };
+
+export const getInputIconWidth = (size) => {
```

Comment:
do we need this function if these values are defined in the object above already?

Reply:
Good point, I rewritten existed function but probably it is not needed